rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -198,20 +198,21 @@
 
     private BridgeSupport bridgeSupport;
 
-    public Bridge(RskAddress contractAddress, BridgeConstants bridgeConstants, BlockchainNetConfig blockchainNetConfig) {
+    public Bridge(RskAddress contractAddress, Constants constants, ActivationConfig activationConfig) {
         this.contractAddress = contractAddress;
-        this.bridgeConstants = bridgeConstants;
-        this.blockchainNetConfig = blockchainNetConfig;
+        this.constants = constants;
+        this.bridgeConstants = constants.getBridgeConstants();
+        this.activationConfig = activationConfig;
     }
 
     @Override
     public long getGasForData(byte[] data) {
-        if (!blockchainConfig.isRskip88() && BridgeUtils.isContractTx(rskTx)) {
+        if (!activationConfig.isActive(ConsensusRule.RSKIP88, rskExecutionBlock.getNumber()) && BridgeUtils.isContractTx(rskTx)) {
```

Comment:
Same goes for this class, 
getActivationConfig().forBlock(rskExecutionBlock.getNumber()).isActive(ConsensusRule.RSKIP88)